src/jenkins_job_insight/notifications.py

```
@@ -0,0 +1,125 @@
+        ):
+            logger.info(
+                "send_mention_notifications: stale subscription (410) for %s, scheduling removal",
+                sub["username"],
+            )
+            return sub["endpoint"]
+        logger.warning(
+            "send_mention_notifications: failed to send to %s: %s",
+            sub["username"],
+            exc,
+        )
+    except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)